web-registry/src/pages/EntityDisplay/EntityDisplay.tsx

```diff
@@ -122,28 +119,19 @@ const EntityDisplay: React.FC<React.PropsWithChildren<unknown>> = () => {
       // https://github.com/regen-network/regen-registry/issues/554
     }
   }
-
-  const Form = (): JSX.Element => (
```

Comment:
this was causing the issue described in regen-network/regen-registry#1413
causing the form to rebuild after submitting hence reinitializing the form values